beets/importer.py

```diff
@@ -675,10 +688,16 @@ def find_duplicates(self, lib):
 
         duplicates = []
         task_paths = {i.path for i in self.items if i}
-        duplicate_query = dbcore.AndQuery((
-            dbcore.MatchQuery('albumartist', artist),
-            dbcore.MatchQuery('album', album),
-        ))
+        keys = config['import']['duplicate_keys'].as_str().split()
+        info = self.chosen_info().copy()
+        info['albumartist'] = artist
```

Comment:
It's not entirely clear to me here why `info` needs to be updated with the results from `chosen_ident`… maybe this should be done inside `chosen_info`? In fact, maybe we want to remove `chosen_ident` altogether if we're replacing it with `chosen_info`?